lighthouse-viewer/package.json

```diff
@@ -28,7 +33,7 @@
     "run-sequence": "^1.1.5",
     "streamqueue": "^1.1.1",
     "tsify": "^2.0.3",
-    "typescript": "^2.1.4",
+    "typescript": "2.9.1-insiders.20180516",
```

Comment:
does it need it's own, can we share with root?

Reply:
> does it need it's own, can we share with root?

have to do `"../node_modules/.bin/tsc -p ."`, but that's not so bad (and probably easier than remembering to keep them in sync). Done